### DISCUSSION TITLE
Custom Cesium Map Chart

### DISCUSSION THREAD

**Discussion by joshk326** [apache/superset#28607] [Q&A / Help]
I'm wanting to create a custom chart which utilizes a cesium map and plots points based on lat long data pulled from a dataset. I was curious if anyone knew of a good way to do this. I have not been very lucky when it comes to finding information regarding the creation of custom charts for superset.

I do know there is the MapBox geojson map which does exactly what I want it to do, but I am needing my dashboards to run completely local and so I am unable to access the MapBox api.

**Comment by joshk326** [comment]
Just confirmed that the geojson map is not by MapBox but rather is a deck.gl map. For some reason Superset has an issue when trying to import my json data to display this chart.

The error is the following:
`Unexpected error
Expecting property name enclosed in double quotes: line 1 column 2 (char 1)`

It seems Superset keeps changing all the double quotes to single quotes when importing the data which then causes this error.

**Reply by imkonsowa** [reply]
@joshk326 Have you ever managed to get geojson working with deck.gl without Mapbox?
